packages/blade/src/utils/getColorScheme/__tests__/getColorScheme.native.test.ts

```diff
@@ -0,0 +1,37 @@
+import getColorScheme from '../../getColorScheme';
+
+jest.mock('react-native/Libraries/Utilities/Appearance', () => ({
```

Comment:
We mock such things in a common place for platoon so that we don't need to do this over and over again

https://github.com/razorpay/frontend-commander/blob/master/commander-platoon/jest-setup.js

Reply:
Yes but for now I just have this mock being used at this place itself hence didn't find a use case to abstract it out